package.json

```diff
@@ -23,6 +23,7 @@
       "import": "./dist/module.es.js"
     },
     "./": "./",
```

Comment:
Shall we get rid of this too?
```suggestion
```

Reply:
If this is ok then yes! I just see that LTS (https://nodejs.org/en/about/releases/) it is supported right now so it is viable to give some time. I don't know how much people use v12 and this library